chart/k8gb/templates/metrics-service.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.k8gb.serviceMonitor.enabled -}}
```

Comment:
we might want to provider the ability to expose metrics with service without enabling ServiceMonitor below(not everybody are using prometheus operator, but still might want to scrape the metrics )

Reply:
@ytsarev , thx for comment. Something like this would make sense than: 

```yaml
`{{- if or .Values.k8gb.exposeMetrics .Values.k8gb.serviceMonitor.enabled -}}`
```

Reply:
@ytsarev @k0da  - I added `exposeMetrics` property.

Reply:
👍